OWNERS

```diff
@@ -1,7 +1,13 @@
 approvers:
   - leogr
+  - zuc
+  - maxgio92
+  - fededp
+  - cpanato
 reviewers:
   - mfdii
```

Comment:
```suggestion
```

We should remove `mfdii` since is inactive for more than a year.

Reply:
Agreed.